pandas/core/internals/managers.py

```diff
@@ -349,14 +349,20 @@ def operate_blockwise(self, other: "BlockManager", array_op) -> "BlockManager":
         """
         return operate_blockwise(self, other, array_op)
 
-    def apply(self: T, f, align_keys=None, **kwargs) -> T:
+    def apply(
+        self: T, f, align_keys=None, ignore_failures: bool = False, **kwargs
+    ) -> T:
         """
         Iterate over the blocks, collect and create a new BlockManager.
 
         Parameters
         ----------
         f : str or callable
             Name of the Block method to apply.
+        align_keys: List[str] or None, default None
```

Comment:
should `align_keys` be typed in the signature too?

Reply:
good idea, updated